src/TextForEvent.tsx

```diff
@@ -33,14 +33,83 @@ import { ROOM_SECURITY_TAB } from "./components/views/dialogs/RoomSettingsDialog
 
 import { logger } from "matrix-js-sdk/src/logger";
 import { removeDirectionOverrideChars } from 'matrix-js-sdk/src/utils';
+import CallHandler from './CallHandler';
+import { RoomMember } from 'matrix-js-sdk/src/models/room-member';
+
+/*
```

Comment:
jsdoc wants two asterisks because it's picky
```suggestion
/**
```